### PR TITLE
Feature/13678 dk

### PR DESCRIPTION
**Pull request by dkmsft**
Thank you for your pull request!
By completing this pull request, you agree to the [Contributing License Agreement](https://github.com/ALM-Rangers/VSTS-Bot/blob/master/.github/CLA.md).

Changes proposed in this pull request:  
- Feature https://almrangers.visualstudio.com/ALM/_workitems/edit/13678 has been implemented.
- A disconnect dialog was added
- In disconnect dialog a RemoveValue was added to remove "userData"

@ALM-Rangers/VSTS-Bot

### CODE REVIEW

File: src/VSTS-Bot.Api/Cards/MainOptionsCard.cs [hunk 1]
```diff
@@ -32,7 +32,8 @@ public MainOptionsCard(bool isConnected)
                 var button2 = new CardAction(ActionTypes.ImBack, Labels.Builds, value: "builds");
                 var button3 = new CardAction(ActionTypes.ImBack, Labels.Connect, value: "connect");
                 var button4 = new CardAction(ActionTypes.ImBack, Labels.Releases, value: "releases");
-                this.Buttons = new List<CardAction> { button1, button2, button3, button4 };
+                var button5 = new CardAction(ActionTypes.ImBack, Labels.Disconnect, value: "Disconnect");
```
Comment (jeffrey-opdam) [inline src/VSTS-Bot.Api/Cards/MainOptionsCard.cs]: Can you make this lower case.

File: src/VSTS-Bot.Api/Dialogs/DisconnectDialog.cs [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+    using System.Threading.Tasks;
+    using Cards;
+    using Microsoft.Bot.Builder.Dialogs;
+    using Microsoft.Bot.Connector;
+    using Resources;
+
+    /// <summary>
+    /// Represents the dialog for disconnecting to a VSTS account and team project.
+    /// </summary>
+    [CommandMetadata("Disconnect")]
```
Comment (jeffrey-opdam) [inline src/VSTS-Bot.Api/Dialogs/DisconnectDialog.cs]: Can you make this lower case?

**Comment by jeffrey-opdam** [comment]
David, can you add some unit tests.

**Comment by dkmsft** [comment]
Jeffrey, I added unittests for the DisconnectDialog, fixed the case of the "d" and updated some unit tests for the MainCard.

File: src/VSTS-Bot.Api/Cards/MainOptionsCard.cs (hunk 1 above)
Comment (dkmsft) [reply src/VSTS-Bot.Api/Cards/MainOptionsCard.cs]: fixed

File: src/VSTS-Bot.Api/Dialogs/DisconnectDialog.cs (hunk 2 above)
Comment (dkmsft) [reply src/VSTS-Bot.Api/Dialogs/DisconnectDialog.cs]: fixed

**Comment by jeffrey-opdam** [comment]
Tnx David. Will you merge the PR?

**Pull request merged**